spec/factories/questions.rb

```diff
@@ -3,7 +3,7 @@
     title { Faker::StarWars.quote }
     content { Faker::Hipster.paragraph }
     views { rand(5) }
-    user
+    user factory: :user
```

Comment:
Now I'm even more curious, because the earlier implemenation automatically assigns a user if none is supplied. Could we pair on this or maybe [check this out](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#aliases); I seem to unable to reproduce your concern. Here's the output of the previous implemenation:
<img width="639" alt="screen shot 2016-04-04 at 7 09 08 pm" src="https://cloud.githubusercontent.com/assets/11221027/14258109/d3ada612-fa98-11e5-8380-b8a7f0222865.png">
It works as expected.